src/Components/benchmarkapps/Wasm.Performance/TestApp/wwwroot/benchmarks/jsonHandling.js

```diff
@@ -16,29 +16,59 @@ group('JSON handling', () => {
   teardown(() => app.dispose());
 
   benchmark('Serialize 1kb', () =>
-    benchmarkJson(app, '#serialize-small', '#serialized-length', 935));
+    benchmarkJson(app, '#serialize-small', '#serialized-length', 935), {
```

Comment:
@SteveSandersonMS prior to this change, we were using the value of the `name` argument (in this case `'Serialize 1kb'`) as the names recorded by the benchmarking infrastructure. @sebastienros suggested we use less prosaic text and follow the pattern that the rest of the infrastructure uses. I took what was admittedly the easiest way to tack on additional content to a `benchmark`. 

I'm happy with this change as is given that it's ultimately test infrastructure, but I'm open to changes if you feel there's a better way to approach this.

Reply:
This is the content we ultimately want to produce: https://gist.github.com/pranavkm/387f83d25600b4d30f7bfc52010227e3